client/src/Components/Common/Header/Header.jsx

```diff
@@ -22,6 +26,10 @@ export default ({
         />
         <Logo showlogo={showlogo} />
       </Grid>
+      <Grid item xs={2} sm={1}>
+        <BurgerMenu showMeun={showMeun} isLogin={isLogin} isAdmin={isAdmin} />
+      </Grid>
+      <Grid item xs={false} sm={2} />
```

Comment:
Is `xs={false}` valid? What does it do?

Reply:
yes  to hide the item on small screens ,  it from Grid material ui documentation